lib/facter/util/file_helper.rb

```diff
@@ -22,16 +22,6 @@ def safe_readlines(path, default_return = [])
           default_return
         end
 
-        def dir_children(path)
-          children = if RUBY_VERSION.to_f < 2.5
-                       Dir.entries(path).reject { |dir| ['.', '..'].include?(dir) }
-                     else
-                       Dir.children(path)
-                     end
-
-          children
-        end
-
```

Comment:
Since this is a public method and callable by custom facts, I think we need to keep the method and just have it call `Dir.children(path)`